dm_cli/utils/utils.py

```diff
@@ -59,58 +57,36 @@ def document_already_exists(api_exception: ApiException) -> bool:
     return False
 
 
-def upload_blobs_in_document(document: dict, data_source_id: str) -> dict:
-    """
-    Uploads any 'system/SIMOS/Blob' types in the document, and replaces the data with UUIDs
-    """
-    try:
-        if document["type"] == SIMOS.BLOB.value:
-            blob_id = document.get("_blob_id", str(uuid4()))
-            blob_name = Path(document["name"]).stem
-            file_like = io.BytesIO(document["_blob_data_"])
-            file_like.name = blob_name
-            res = dmss_api.blob_upload(data_source_id, blob_id, file_like)
-            return {
-                "name": blob_name,
-                "type": SIMOS.BLOB.value,
-                "_blob_id": blob_id,
-                "size": len(document["_blob_data_"]),
-            }
-    except KeyError as error:
-        reduced_document = {k: v for k, v in document.items() if isinstance(v, str)}
-        raise KeyError(f"The document; '{reduced_document}' is missing a required attribute: {error}")
-    except ApiException as api_exception:
-        if document_already_exists(api_exception):
-            return {
-                "name": blob_name,
-                "type": SIMOS.BLOB.value,
-                "_blob_id": blob_id,
-                "size": len(document["_blob_data_"]),
-            }
-        raise
+def add_package_to_path(name: str, path: Path):
+    package = Package(name, is_root=len(path.parts) == 2)
+    dmss_api.document_add(
+        str(path.parent),
+        json.dumps(package.to_dict()),
+        update_uncontained=False,
+        files=[],
+    )
+
 
+def replace_file_addresses(document: dict, data_source_id: str, files_to_upload) -> dict:
+    if document["type"] == SIMOS.REFERENCE.value:
+        if document["address"] in files_to_upload:
+            blob_id = files_to_upload[document["address"]]
+            # print(f"\tReplacing {document['address']} with ${blob_id}")
+            document["address"] = f"${blob_id}"
```

Comment:
```suggestion
    if document["type"] == SIMOS.REFERENCE.value and document["address"] in files_to_upload:
          blob_id = files_to_upload[document["address"]]
          document["address"] = f"${blob_id}"
```

can avoid nested if here, also can remove print line